gateway/coprocess_bundle.go

```diff
@@ -425,7 +426,7 @@ func (gw *Gateway) loadBundle(spec *APISpec) error {
 		if err := os.RemoveAll(bundle.Path); err != nil {
 			bundleError(spec, err, "Couldn't remove bundle")
 		}
-		return nil
+		return err
```

Comment:
@andrei-tyk I think you have to add return err into the conditional block above. I think this may return the previous value out of scope, since if redeclares err